### PR TITLE
Update Onboarding Worflow

### PR DESCRIPTION
**Pull request by elfredawong**
Adding clear guidelines about the support ticket workflow for network onboarding

### CODE REVIEW

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
+Here's a summary of steps to join Corda Network. Before you proceed with the steps in this page, make sure that you have read and understood the following documentations:
 
 1. [Pre-requisite to join](https://corda.network/joining-corda-network/pre-requisites)
 2. [Corda Network Rulebook](https://corda.network/corda-network-rulebook/introduction) 
 3. [Corda Network SLA Handbook](https://corda.network/trust-centre/sla-&-maintenance-handbook)
 
 ## Step 1: Introductory Call and Terms of Use Agreement
 
-If you are decide to join Corda Network, you should send an email to info@corda.network. You should answer the following questions:
+If you are interested to join Corda Network, please send an email to info@corda.network with answers to these questions:
```
Comment (KadriSiff) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:11]: If we are pushing the Contact Us form onto the website, we can remove the ask to email info@corda.network

Comment (elfredawong) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md:11]: Yes good point - we can edit this point once the contact us form is available on the production website

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
 
-"O" - Organisation should usually be the same Legal Entity as what you have filled out in your Terms of Use (Step 1), which is also the owner of the assets being used on Corda Network. For guidance on which legal entity name to pick for your node, see the [Allowable Identity Names](https://corda.network/corda-network-rulebook/allowable-identity-names-on-corda-network) session.
+i. You will raise a [Task](https://r3-cev.atlassian.net/servicedesk/customer/portal/2/group/2/create/79) ticket on the Corda Network Support Portal. *Note: Your access to the Corda Network Support Portal will be granted after you signed the Corda Network Terms of Use, and your email address needs to be filled out on the Contact Details table on the Terms of Use. If you have any questions or needs access, please contact your account manager.*
```
Comment (KadriSiff) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:26]: after signing

Comment (KadriSiff) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:26]: "or needs access" doesn't make sense

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 3]
```diff
@@ -1,29 +1,35 @@
...
+
+ii. Title the ticket "Pre-approval of X.500 name", copy the following 4 questions and answer them on the free text field on the ticket. Please note that you need to fill out a separate ticket for each identity that you are adding.
+
+1. What is the X.500 name of your node?
+
+X.500 name is your node's unique identifier on Corda Network. It consists of the following 6 fields. Check out [Allowable Identity Names](https://corda.network/corda-network-rulebook/allowable-identity-names-on-corda-network) session of the Corda Network Rulebook for detailed guidance of naming your X.500.
```
Comment (KadriSiff) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:32]: section

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 4]
```diff
@@ -33,29 +39,35 @@ i. The node operator fills out fields in bold in the table below, for every node
 | **Organisation Unit (OU)**  |     - |
 | State (ST)                  |     - |
 | Common Name (CN)            |     - |
-| Node Operator email address |     - |
-
 
-ii. CNF performs legal identity and sanctions checks on Organisation field (O) for each node.
-
-iii. CNF needs to verify the person submitting this CSR has access to the associated email inbox and will do so by sending an email to the ‘node operator email address’ and waiting for a confirmation reply.
+- "O" Organisation should be the same Legal Entity as what you have filled out in your Terms of Use (Step 1), unless you are onboarding for your sponsee. This is also the owner of the assets being used on Corda Network. For guidance on which legal entity name to pick for your node, see the [Allowable Identity Names](https://corda.network/corda-network-rulebook/allowable-identity-names-on-corda-network) session.
```
Comment (KadriSiff) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:43]: it seems like autocorrect is changing "section" into "session"

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 5]
```diff
@@ -33,29 +39,35 @@ i. The node operator fills out fields in bold in the table below, for every node
...
+- Fields in non-bold are optional, except for State (S), which is required only for legal entities registered in a country with federal states system (e.g. USA, Australia, Canada, etc).
+- The X.500 name is checked for uniqueness. If a CSR is submitted (or a certificate exists) with an X.500 name already in Corda Network, it will be automatically rejected.
 
-iv. The network operator will give the all clear to node operator to submit their CSR.
+2. What is the official registered address of the legal entity?
+3. Are you submitting the node on behalf of your entity, or your sponsee's entity?
+4. Is the legal entity is on a sanction list observed by Corda Network?
 
-### Before you begin...
+- You can refer to Section 1.2 of the [Admission Criteria](https://corda.network/corda-network-rulebook/admissions-criteria/) for the list of sanction lists observed by Corda Network.
```
Comment (KadriSiff) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:53]: sanctions lists (bc lists implies we have multiple ones)

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 6]
```diff
@@ -33,29 +39,35 @@ i. The node operator fills out fields in bold in the table below, for every node
...
 
-- Fields in non-bold are optional, except for State (S), which is required only for legal entities registered in a country with federal states system (e.g. USA, Australia, Canada, etc).
+iii. The Corda Network oeprator (R3 currently) will perform due diligence check on the proposed identity name you submitted. This includes:
```
Comment (KadriSiff) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:55]: operator is misspelled

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 7]
```diff
@@ -33,29 +39,35 @@ i. The node operator fills out fields in bold in the table below, for every node
...
+1. **Legal entity check** - we will check if the legal entity (Organisation field, or "O" field) of the registering node is an officially registered company. *Note: We do not accept individuals joining the Corda Network at the moment.*
+2. **Sanction check** - we will check if the legal entity is on a sanction list observed by the Corda Network operator
+3. **Naming convention check** - we will check if your proposed x500 name follows the all naming convention rules set out in the [Corda Network Rulebook](https://corda.network/corda-network-rulebook/introduction).
 
-- The combination of O and OU are checked for uniqueness. If a CSR is submitted (or a certificate exists) with these two fields already in Corda Network, it will be rejected.
+### Other things to note...
 
-- If you are unsure of how to populate any fields, please see the guidance at the end of the email or reply to this email with your specific questions.
+- If you are setting up a notary, there are additional considerations you need to keep in mind, please check [Customer Hub](https://customerhub.r3.com/) for instructions to set up a notary.
```
Comment (KadriSiff) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:63]: afaik the notary set up instructions are not there. The only public-facing document is this> https://docs.corda.net/docs/corda-enterprise/4.8/corda-network/setup-segregated-notary.html#setting-up-a-notary-in-a-segregated-network

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 2 above)
Comment (elfredawong) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md:26]: Updated

Comment (elfredawong) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md:26]: What about changing it to "want to request access"?

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 3 above)
Comment (elfredawong) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md:32]: Updated

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 4 above)
Comment (elfredawong) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md:43]: Updated

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 6 above)
Comment (elfredawong) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md:55]: Updated!

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 1 above)
Comment (carolynequinn) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md:11]: I think this whole step would change quite a lot once that form is ready

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 2 above)
Comment (carolynequinn) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md]: i would just keep as 'if you have any questions'

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 8]
```diff
@@ -140,23 +156,37 @@ Start polling server for certificate signing approval.
...
 
-Please also ensure that the relevant lists are being subscribed - each subscriber can choose which service(s) (notary, network map and identity operator) and enviornement(s) (Pre-production and Production) to subscribe to. **We strongly recommend that at least the node operator email address should subscribe to all services in your operating environment. You should also ensure you update your subscription list when you onboard to a new environment - such as ensuring you subscribe to all Production updates of as you onboarded to the Production environment).**
+Please also ensure that the relevant lists are being subscribed - each subscriber can choose which service(s) (notary, network map and identity operator) and enviornement(s) (Pre-production and Production) to subscribe to. **We strongly recommend that at least one individual in your organisation should subscribe to all services in your operating environment. You should also ensure you update your subscription list when you onboard to a new environment - such as ensuring you subscribe to all Production updates of as you onboarded to the Production environment).**
```
Comment (carolynequinn) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:190]: Please also ensure that the relevant lists are being subscribed **to**

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 6 above)
Comment (carolynequinn) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md]: operator not oeprator

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 9]
```diff
@@ -140,23 +156,37 @@ Start polling server for certificate signing approval.
...
+
+*Hi Corda Network team,*
+
+*Please approve the CSR submitted for Production / Pre-production (choose one) environment.*
+
+- *X.500 Name:*
+- *Request ID:*
+- *Link to pre-approval ticket:*                  
+
+A member of the team will review the ticket and approve your CSR accoridngly.
```
Comment (carolynequinn) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:172]: accordingly

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 10]
```diff
@@ -140,23 +156,37 @@ Start polling server for certificate signing approval.
 
 Important: the Request ID given in the above should be noted and kept safe for future reference.
```
Comment (carolynequinn) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:158]: why? would be good if we can explain more clearly why they should keep the Request ID

File: content/3.Joining Corda Network/4.Onboarding Workflow.md [hunk 11]
```diff
@@ -1,29 +1,35 @@
...
-If you are decide to join Corda Network, you should send an email to info@corda.network. You should answer the following questions:
+If you are interested to join Corda Network, please send an email to info@corda.network with answers to these questions:
 
 1. Are you building / running a Corda application, or looking to use a Corda application?
 2. What is the use case and name (if already available) of your Corda application?
 3. Are you using Corda Open Source or Enterprise?
 4. Have you been in touch with any other R3 personnel?
 
-A member of our team will schedule a call with you to go through the onboarding process. We can also advise you on the network choice and contracting model. You can find more information about which Terms of Use agreement is most suitable for you on the Contracting Model session. 
+A member of our team will schedule a call with you to go through the onboarding process. We will also advise you on the network choice and contracting model.
```
Comment (carolynequinn) [inline content/3.Joining Corda Network/4.Onboarding Workflow.md:18]: An R3 representative will schedule a call with you

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 6 above)
Comment (carolynequinn) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md]: due diligence checks

**Review by carolynequinn — CHANGES_REQUESTED** [review]

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 1 above)
Comment (elfredawong) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md:11]: Agree - this should be in place very shortly

File: content/3.Joining Corda Network/4.Onboarding Workflow.md (hunk 10 above)
Comment (elfredawong) [reply content/3.Joining Corda Network/4.Onboarding Workflow.md]: added more details to this

**Review by elfredawong — COMMENTED** [review]
Updated per suggestion!

**Pull request merged**